### DISCUSSION TITLE
如果能够支持导入epub的笔记就好了

### DISCUSSION THREAD

**Discussion by xiaoqiangch** [aidenlx/obsidian-zotero#239] [Ideas]
现在zotero 7已经支持epub高亮，希望也能够直接导入epub文件中的笔记！

**Comment by aidenlx** [comment]
v2 支持 EPUB 标注。

EPUB 没有 PDF 那样的页码，所以 `zt.page` 是空的；其余字段（文本、批注、颜色、标签、跳转链接）和 PDF 标注完全一致，annotation 模板不用为 EPUB 单独改。

v2 beta 现已发布：https://zotlit.aidenlx.site/docs/install-zotlit
迁移指南：https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
